src/main/java/io/github/railroad/config/SyntaxConfig.java

```diff
@@ -48,7 +45,7 @@ public SyntaxConfig() {
 				JSONObject obj = new JSONObject(tokenizer);
 				JSONArray rules = obj.getJSONArray("rules");
 
-				Map<String, EnumSyntaxType> ruleMap = new HashMap<>();
+				Map<String, EnumSyntaxType> ruleMap = new HashMap();
```

Comment:
IJ's gonna yell at me if you make it raw >:(
IJ scares me man 🤣

Reply:
i mean. just turn off the raw types warning lol

Reply:
I could... but the warning is there for a reason :wink:

Reply:
and you are just trying to bypass that reason by doing `<>` xd

Reply:
It's not bypassing. Including it tells java to look at the declaration.